Installer.cpp

```diff
@@ -222,14 +224,32 @@ HRESULT NppShell::Installer::RegisterSparsePackage()
 
     options.ExternalLocationUri(externalUri);
 
-    auto deploymentOperation = packageManager.AddPackageByUriAsync(packageUri, options);
+    auto deploymentOperation = packageManager.StagePackageByUriAsync(packageUri, options);
     auto deployResult = deploymentOperation.get();
 
     if (!SUCCEEDED(deployResult.ExtendedErrorCode()))
     {
         return deployResult.ExtendedErrorCode();
     }
 
+    Package package = GetSparsePackage();
+    if (package == NULL)
+    {
+        return S_FALSE;
+    }
+
+    winrt::hstring familyName = package.Id().FamilyName();
+
+    deploymentOperation = packageManager.ProvisionPackageForAllUsersAsync(familyName);
+    deployResult = deploymentOperation.get();
+
+    if (!SUCCEEDED(deployResult.ExtendedErrorCode()))
+    {
+        return deployResult.ExtendedErrorCode();
+    }
+
+    SHChangeNotify(SHCNE_ASSOCCHANGED, SHCNF_IDLIST, 0, 0);
```

Comment:
I think it is not needed since it is already done by the install method

Reply:
You are right, I'll remove it later and update the PR

Reply:
Done 🙂